LeapingGorilla.Testing.XUnit/LeapingGorilla.Testing.XUnit.csproj

```diff
@@ -24,11 +23,12 @@ From these needs LeapingGorilla.Testing was born.</Description>
         <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
         <PackageId>LeapingGorilla.Testing.XUnit</PackageId>
         <Product>LeapingGorilla.Testing.XUnit</Product>
+        <TargetFrameworks>net8.0;net6.0;netstandard2.0</TargetFrameworks>
```

Comment:
We can drop net6 here

Reply:
net6 is still in support until November this year. I'm pretty sure net8 assemblies can't be referenced by a 6 codebase. They could still reference the netstandard version I guess?